resources/testdata/glyphs2/OneGlyph.glyphs

```diff
@@ -0,0 +1,32 @@
+{
+familyName = "New Font";
+fontMaster = (
+    {
+        id = m01;
+    }
+);
+glyphs = (
+{
+    glyphname = period;
```

Comment:
might as well call this '.notdef', which is a required glyph anyway

Reply:
I like it, so updated

Reply:
thanks. This reminds me of https://github.com/googlefonts/fontmake-rs/issues/149